srcpkgs/apparmor/files/profiles/usr.bin.dhcpcd

```diff
@@ -26,9 +32,9 @@ profile dhcpcd /{usr/,}bin/dhcpcd {
   /proc/sys/net/ipv{4,6}/neigh/*/retrans_time_ms w,
   /proc/sys/net/ipv{4,6}/neigh/*/base_reachable_time_ms w,
 
-  /{var/,}run/dhcpcd{-*,}.pid rwk,
-  /{var/,}run/dhcpcd.sock rw,
-  /{var/,}run/dhcpcd.unpriv.sock rw,
+  /{var/,}run/dhcpcd/pid rwk,
```

Comment:
This can be `eth0.pid` or similar, depending on how dhcpcd is configured. You should change it to:

```
  /{var/,}run/dhcpcd/{,*.}pid rwk,
```